src/main/kotlin/no/fdk/fdk_concept_harvester/adapter/FusekiAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package no.fdk.fdk_concept_harvester.adapter
+
+import no.fdk.fdk_concept_harvester.configuration.FusekiProperties
+import no.fdk.fdk_concept_harvester.rdf.createRDFResponse
+import org.apache.jena.rdf.model.Model
+import org.apache.jena.riot.Lang
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import java.io.OutputStreamWriter
+import java.net.HttpURLConnection
+import java.net.URL
+
+private val LOGGER = LoggerFactory.getLogger(FusekiAdapter::class.java)
```

Comment:
Hadde foretrukket `@Sl4j` fra lombok i stedet for den

Reply:
Har ikke planer om å dra inn lombok kun for `@Sl4j` :P